src/containers/NFT/NFTHeader/Details.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { CopyToClipboard } from '../../shared/components/CopyToClipboard'
+import './styles.scss'
+import { formatPrice } from '../../shared/utils'
+
+interface Props {
+  data: {
+    NFTId: string
+    ledgerIndex: number | undefined
+    owner: string
+    isBurned: boolean | undefined
+    flags: string[] | undefined
+    transferFee: number
+    issuer: string
+    NFTTaxon: number
+    NFTSequence: number | undefined
+    uri: string | undefined
+    validated: boolean | undefined
+    status: string | undefined
+    warnings: string[] | undefined
+    minted: string | undefined
+    domain: string | undefined
+    emailHash: string | undefined
+  }
+}
+
+const Details = ({ data }: Props) => {
+  const { minted, domain, emailHash, NFTTaxon, uri, transferFee } = data
+  const { t } = useTranslation()
+
+  const formattedFee =
+    transferFee === 0
+      ? `${transferFee}%`
+      : `${formatPrice(transferFee / 1000, {
+          currency: 'none',
+          decimals: 5,
+          padding: 3,
+        })}%`
+
+  const abbrvEmail =
+    emailHash &&
+    (emailHash.length > 20 ? emailHash.slice(0, 20).concat('...') : emailHash)
+  const abbrvURI =
+    uri && (uri.length > 20 ? uri.slice(0, 20).concat('...') : uri)
+
+  return (
+    <table className="token-table">
+      <tbody>
+        {minted && (
+          <tr className="row">
+            <td className="col1">{t('minted')}</td>
+            <td className="col2">{minted}</td>
+          </tr>
+        )}
+        {domain && (
+          <tr className="row">
+            <td className="col1">{t('domain')}</td>
+            <td className="col2">
+              <a
+                href={`https://${domain}`}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {domain}
+              </a>
+            </td>
+          </tr>
+        )}
+        {emailHash && (
+          <tr className="row">
+            <td className="col1">{t('email_hash')}</td>
+            <td className="col2">
+              {abbrvEmail}
+              <CopyToClipboard className="copy" text={emailHash} />
+            </td>
+          </tr>
+        )}
+        <tr className="row">
+          <td className="col1">{t('taxon_id')}</td>
+          <td className="col2">{NFTTaxon}</td>
+        </tr>
+        {uri && (
+          <tr className="row">
+            <td className="col1">{t('uri')}</td>
+            <td className="col2">
+              <a href={uri} target="_blank" rel="noopener noreferrer">
+                {abbrvURI}
+              </a>
```

Comment:
@ledhed2222 isn't it a security issue to have external links?

Reply:
We currently like the domain for validators and I think account pages.  I do have a PR open that will add an external link icon to all external links.  That could be enough.

Reply:
I can pull out that from https://github.com/ripple/explorer/pull/240.

Reply:
It is now in a separate pr #247

Reply:
yes, i think we should not link out for now. doing so invites all kinds of potential security issues that we haven't thought through enough. we can always add this back